skynet/modules/stt/streaming_whisper/utils/utils.py

```diff
@@ -196,7 +196,7 @@ def convert_seconds_to_bytes(cut_mark: float) -> int:
     return int(cut_mark / cfg.one_byte_s)
 
 
-def is_silent(audio: bytes) -> Tuple[bool, iter]:
+async def is_silent(audio: bytes) -> Tuple[bool, iter]:
```

Comment:
I see no await in there, what's async about it?

Reply:
leftover